CLAUDE.md

```
@@ -0,0 +1,240 @@
+cd tests/e2e-playwright && npx playwright test     # E2E
+```
+
+**Run a single test:**
+- Codeception: `vendor/bin/codecept run unit Path/To/TestCest.php:methodName`
+- Playwright: `cd tests/e2e-playwright && npx playwright test path/to/spec.ts -g "test name"`
+
+---
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)